javascript/javascript3/week3/homework-js3-week3/users.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const BASE_URL_CRUD_API2 = `https://crudcrud.com/api/${API_KEY_CRUD_API}`;
+
+let user = null;
+
+const loginButton = document.getElementsByClassName('login-button')[0];
```

Comment:
Even though getElementsByClassName could be beneficial, for example, if you want to iterate over all the array of selected elements and do something with them, in this specific case it looks like a more appropriate selector would be getElementsByClass or getElementsById.